src/test/scala/service/WrapperServiceSpec.scala

```diff
@@ -197,6 +201,27 @@ class WrapperServiceSpec extends AsyncWordSpec with Matchers with MockitoSugar w
       disableSessionExpiredCaptor.getValue mustBe disableSessionExpired
       contentCaptor.getValue mustBe content
     }
+
+    "return the continueUrl if it is a valid relative URL" in {
+      val continueUrl = Some(RedirectUrl("/url"))
+      val expectedUrl = "/url"
+
+      val actualUrl = wrapperService.safeSignoutUrl(continueUrl)
+
+      actualUrl mustBe Some(expectedUrl)
+    }
+
+    "return the exitSurveyOrigin if the continueUrl is None" in {
+      val continueUrl = None
+      implicit val appConfg: AppConfig = injector.instanceOf[AppConfig]
```

Comment:
In wrapperservicespec, we mocked the dependencies like appconfig in other unit tests. but in this test, appconfig is coming from dependency injection. Should we also use mocked one ? 
we can test appconfig logic independently ?